rmf_traffic/src/rmf_traffic/schedule/Negotiation.cpp

```diff
@@ -1296,7 +1323,7 @@ auto Negotiation::evaluate(const Evaluator& evaluator) const -> ConstTablePtr
 
     const auto& proposal = Table::Implementation::get(*table_ptr).proposal;
     assert(Table::Implementation::get(*table_ptr).itinerary);
-    assert(!Table::Implementation::get(*table_ptr).rejected);
+    assert(!*Table::Implementation::get(*table_ptr).rejected);
```

Comment:
I think maybe you haven't tried compiling the latest changes in debug mode, because this should produce a compilation error when `NDEBUG` is not defined.

Reply:
fixed a7c6ccb